webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
This one makes sense I think, though I observe that elsewhere (like for ICE gathering state directly below) we tend to queue a task to set the state *and* fire the event, though here it may matter more that the state is set early? Should we worry that the state may have changed again by the time the event fires? I'm unsure.